modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/session.es.js

```diff
@@ -16,7 +16,7 @@ import fetch from './fetch.es';
 
 const TOKEN_SERIALIZE = 'serialize://';
 
-function getSessionClickFormData(cmd) {
+function getSessionClickFormData(cmd, options) {
```

Comment:
`options` could be anything in the future, so in order to be more explicit here, I think we could destructure only the relevant parts here like:

```javascript
function getSessionClickFormData(cmd, {useHttpSession}) {
    [...]

    if (useHttpSession) {
        formData.append('useHttpSession', true);
    }
}
```

Reply:
This way, `options` remains a bag of options that can have different purposes and affect the request in different ways.

For example, one option could add some headers rather than form data and we would be able to implement it without those new options bleeding through the data.

Reply:
@antonio-ortega, @jcampoy, just confirming wether you want us to pick this up or if you'll be working on it.